MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -387,6 +387,18 @@
 				formatFeatures->formatA4B4G4R4 = canSupport4444;
 				break;
 			}
+			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_EXTENDED_DYNAMIC_STATE_FEATURES_EXT: {
+				auto* extDynState = (VkPhysicalDeviceExtendedDynamicStateFeaturesEXT*)next;
+				extDynState->extendedDynamicState = true;
```

Comment:
Can we advertise support for this prior to Metal 3.1? It looks like dynamic vertex strides are required by this feature.

Reply:
Unlike DS3's`VkPhysicalDeviceExtendedDynamicState3FeaturesEXT `, `VkPhysicalDeviceExtendedDynamicStateFeaturesEXT` only has a single enablement flag, so the entire extension has a binary enablement.

Users had been asking for `VK_EXT_extended_dynamic_state`, knowing that Metal did not support dynamic vertex stride, and I figured it would be one of the least-used dynamic states anyway, so I figured it wouldn't be right to limit the entire extension based on vertex stride limitations.

When it comes to it, we'll have to introduce a portability flag. In the meantime, the limitation is documented.